packages/core/admin/admin/src/hooks/useFetchInstalledPlugins/index.js

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from 'react-query';
+import { useIntl } from 'react-intl';
+import { useNotifyAT } from '@strapi/design-system/LiveRegions';
+import { useNotification } from '@strapi/helper-plugin';
+import { fetchInstalledPlugins } from './utils/api';
+
+const useFetchInstalledPlugins = title => {
+  const toggleNotification = useNotification();
+  const { formatMessage } = useIntl();
+  const { notifyStatus } = useNotifyAT();
+
+  const notifyLoad = () => {
+    notifyStatus(
+      formatMessage(
+        {
+          id: 'app.utils.notify.data-loaded',
+          defaultMessage: 'The {target} has loaded',
+        },
+        { target: title || 'Plugins' }
```

Comment:
This would read "The Plugins has loaded". Should "Plugin" be singular? Or should it be "Marketplace" instead?

Reply:
I think it should still be plugins but I should update `has` to `have`

Reply:
Actually you might be right, taking a second look

Reply:
I'm wondering if we even need a notify load for installed plugins. If we notify the Marketplace has loaded then shouldn't that be enough?